moped-etl/prefect/tasks/database.py

```diff
@@ -1,13 +1,20 @@
 import os
-import prefect
 
+import prefect
 from prefect import task
+from prefect.tasks.shell import ShellTask
+
 import psycopg2
 from psycopg2.extensions import ISOLATION_LEVEL_AUTOCOMMIT
 
 DATABASE_HOST = os.environ["MOPED_TEST_HOSTNAME"]
 DATABASE_USER = os.environ["MOPED_TEST_USER"]
 DATABASE_PASSWORD = os.environ["MOPED_TEST_PASSWORD"]
+MOPED_READ_REPLICA_HOST = os.environ["MOPED_READ_REPLICA_HOST"]
+MOPED_READ_REPLICA_USER = os.environ["MOPED_READ_REPLICA_USER"]
+MOPED_READ_REPLICA_PASSWORD = os.environ["MOPED_READ_REPLICA_PASSWORD"]
+
+MOPED_READ_REPLICA_HOST
```

Comment:
Straggler here that I think can get the boot.

Reply:
Yes, this definitely needs to go 🥾